hera_cal/vis_clean.py

```diff
@@ -6,6 +6,9 @@
 from collections import OrderedDict as odict
 import datetime
 from uvtools import dspec
+from copy import deepcopy
```

Comment:
you've got `copy` module below, lets just use `copy.deepcopy` instead of re-importing

Reply:
I've cleaned this up.